src/modules/task.js

```diff
@@ -0,0 +1,41 @@
+import listItem from './list-item.js';
+import TaskStorage from './task-storage.js';
+
+export default class Task {
+  static LIST = TaskStorage.exist() ? TaskStorage.get() : [];
+
+  constructor(description, completed = false) {
+    this.description = description;
+    this.completed = completed;
+    this.index = Task.LIST.length;
+  }
+
+  appendToList() {
+    Task.LIST.push(this);
+  }
+
+  static remove(index) {
+    Task.LIST = [...Task.LIST.filter((x) => x.index !== index)];
+    Task.updateIndexes();
+  }
+
+  static updateIndexes() {
+    let initIndex = 0;
+    Task.LIST.forEach((elmt) => {
+      elmt.index = initIndex;
+      initIndex += 1;
+    });
+  }
+
+  static getTask(id) {
+    return Task.LIST.find((x) => x.index === id);
+  }
+
+  static list() {
+    return Task.LIST.map((item) => listItem(item)).join('');
+  }
```

Comment:
- [ ] Great job implementing 👏🏻 however On page load, the tasks should appear in order of the index values for each task,  I strongly suggest you set the property index to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal 5) the  index number should start with 1, not zero, according to this [project requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#:~:text=By%20default%20new%20tasks%20should%20have%20the%20property%20completed%20set%20to%20false%20and%20the%20property%20index%20set%20to%20the%20value%20of%20the%20new%20array%20length%20(i.e.%20if%20you%27re%20adding%20a%205th%20task%20to%20the%20list%2C%20the%20index%20of%20that%20task%20should%20equal%20to%205).). Example: According to this PR your starting is index 0, but it should be index 1 instead.😄 

- [ ] Don't forget, 🤔 Deleting a task updates all remaining items' indexes, so they represent the current list order and are unique.

- I hope this is more clear now. If you have any questions, please do not hesitate to tag me on @BenMukebo or send me a direct message on slack. :v: 🤗

Reply:
@BenMukebo  Thanks For the feedback I really appreciate :+1:  However I don't really think I left out the second raised issue

Reply:
hello, @bienvenuushindi I think you have already implemented the second raised issue 👇🏻 
> 
> - [ ] Kindly add a list of technology you have used in the project 👨🏻‍🎨 🔢

- Thanks